spec/ddtrace/contrib/sinatra/multi_app_spec.rb

```diff
@@ -69,8 +69,15 @@
 
         it do
           is_expected.to be_ok
-          expect(spans).to have(2).items
+          expect(spans).to have(3).items
```

Comment:
Is the extra span the Rack span?
Does this mean that users upgrading to the code in this PR will see an extra span in their trace?

Reply:
Yes, and yes.